### PR TITLE
Improvements/bal 169 enhance coloring

### PR DESCRIPTION
**Pull request by gperissetcelteeka**
# NEW COLORING SECTION / VISUAL IMPROVEMENTS

## :recycle: Current situation & Problem
*Describe the current situation (if possible, with an exemplary (or real) code snippet and/or where this is used)*
*Please link any open issue that is addressed with this PR*
#169 
#170 

## :bulb: Proposed solution
*Describe the solution and how this affects the project and internal structure*

## :gear: Release Notes 
*Add a summary of the feature and possible migration guides if this is a breaking change so this section can be added to the release notes.*
*Include code snippets that provide examples of the feature implemented if it appends or changes the public interface.*

## :heavy_plus_sign: Additional Information
*Provide some additional information if possible*

### Related PRs
*Reference the related PRs*

### Testing
*Are there tests included? If yes, which situations are tested, and which corner cases are missing?*

### Reviewer Nudging
*Where should the reviewer start? Where is a good entry point?*

### Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md):
- [x] I agree to follow the [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: Balance/BalanceAppDelegate.swift [hunk 1]
```diff
@@ -35,8 +35,6 @@ class BalanceAppDelegate: CardinalKitAppDelegate {
             if HKHealthStore.isHealthDataAvailable() {
                 healthKit
             }
-#else
-            FirebaseAccountConfiguration()
```
Comment (sebastianricaldoni) [inline Balance/BalanceAppDelegate.swift:39]: @gperissetcelteeka just checking we don't need this.

File: Balance/Distraction/Drawing/BackgroundDrawView.swift [hunk 2]
```diff
@@ -69,7 +69,8 @@ struct BackgroundDrawView: View {
                         viewName: "Mandala Highlight Selected: " + mandala.name,
                         isDirectChildToContainer: true,
                         content: {
-                            DrawView(currentDraw: $currentDraw, isNewDraw: true, isColoring: true)
+                            DrawSketch(draw: currentDraw)
+//                            DrawView(currentDraw: $currentDraw, isNewDraw: true, isColoring: true)
```
Comment (sebastianricaldoni) [inline Balance/Distraction/Drawing/BackgroundDrawView.swift:73]:
```suggestion
```

File: Balance/Home/Home.swift [hunk 3]
```diff
@@ -122,11 +118,11 @@ struct HomeView: View {
...
-#if !DEMO
-                let user = firebaseAccountConfiguration.user
-                let isFirstLoadKey = (user?.uid ?? "0") + "isFirstLoad"
-                UserDefaults.standard.set(true, forKey: isFirstLoadKey)
-#endif
+// #if !DEMO
+//                let user = firebaseAccountConfiguration.user
+//                let isFirstLoadKey = (user?.uid ?? "0") + "isFirstLoad"
+//                UserDefaults.standard.set(true, forKey: isFirstLoadKey)
+// #endif
```
Comment (sebastianricaldoni) [inline Balance/Home/Home.swift:125]: @gperissetcelteeka just double checking you wanted to remove this on purpose

File: Balance/Login/LoginViewLocal.swift [hunk 4]
```diff
@@ -118,6 +103,18 @@ struct LoginViewLocal: View {
                 .padding(.horizontal, 20)
             Divider()
                 .padding(.horizontal, 20)
+            HStack {
+                Spacer()
+                Text("Existing a user? ")
```
Comment (sebastianricaldoni) [inline Balance/Login/LoginViewLocal.swift:108]:
```suggestion
                Text("Existing user? ")
```

File: Balance/Profile/UserProfileRepositoryLocal.swift [hunk 5]
```diff
@@ -80,7 +91,7 @@ class UserProfileRepositoryToLocal: ObservableObject {
             return
         }
         
-        UserDefaults.standard.removeObject(forKey: userID)
+//        UserDefaults.standard.removeObject(forKey: userID)
```
Comment (sebastianricaldoni) [inline Balance/Profile/UserProfileRepositoryLocal.swift:94]:
```suggestion
```

**Review by sebastianricaldoni — CHANGES_REQUESTED** [review]
Left some comments and a typo to fix
Looking great though @gperissetcelteeka 🚀

File: Balance/BalanceAppDelegate.swift (hunk 1 above)
Comment (gperissetcelteeka) [reply Balance/BalanceAppDelegate.swift:39]: Its ok

**Review by sebastianricaldoni — APPROVED** [review]
